services/operator_info.go

```diff
@@ -0,0 +1,134 @@
+package services
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Rocket-Rescue-Node/credentials"
+	"github.com/Rocket-Rescue-Node/credentials/pb"
+	"github.com/Rocket-Rescue-Node/rescue-api/models"
+	authz "github.com/Rocket-Rescue-Node/rescue-api/models/authorization"
+	"github.com/Rocket-Rescue-Node/rescue-api/util"
+	"go.uber.org/zap"
+)
+
+const (
+	// The maximum age for a operator info request to be considered valid.
+	operatorInfoRequestMaxAge = time.Duration(15) * time.Minute
+)
+
+type OperatorInfo struct {
+	CredentialEvents []int64 `json:"credentialEvents"`
+	QuotaSettings    *Quota  `json:"quotaSettings,omitempty"`
```

Comment:
I would remove this from this struct and instead use a getter in the handler to populate the response body field.

Reply:
Are you referencing the QuotaSettings here?

Reply:
Yes- Since it's "globally defined" there's no reason to put it in the OperatorInfo.

We only need it on OperatorInfoResponse. We know the operator_type in the handler, so we can ask for a copy of the quotaSettings json from the handler directly. The OperatorInfo service is then "pure" in that it only really returns info about the operator, not info about the global quota.

Reply:
I didn't fully understand the rationale at first, so I appreciate your explanation. Change made!